tools/internal_ci/macos/grpc_interop_toprod.sh

```diff
@@ -30,4 +30,4 @@ export GRPC_DEFAULT_SSL_ROOTS_FILE_PATH="$(pwd)/etc/roots.pem"
 # building all languages in the same working copy can also lead to conflicts
 # due to different compilation flags
 # TODO(jtattermusch): add --cloud_to_prod_auth support
-tools/run_tests/run_interop_tests.py -l c++ --cloud_to_prod -t -j 4
+tools/run_tests/run_interop_tests.py -l c++ --cloud_to_prod --prod_servers default gateway_v4 -t -j 4
```

Comment:
Please use cloud_gateway_v4, which covers Cloudpath.

Reply:
This PR got merged as a side effect of #14985. Let's discuss offline  why cloud_gateway_v4 is the right choice here, I'd like to understand it more.

Reply:
Sorry I forgot this is outside Google Cloud. No need to change. :)